shiny/playwright/controller/_navs.py

```diff
@@ -631,20 +638,30 @@ def expect_position(
                 re.compile(rf"{position}"), timeout=timeout
             )
 
-    def expect_inverse(self, *, timeout: Timeout = None) -> None:
+    def expect_inverse(
+        self,
+        value: bool,
```

Comment:
This new parameter will need a changelog entry

Also:
* `.expect_layout()` -> `.expect_fluid()`
* Added `PageNavbar` class